Posts/2021/07/tfl-regkey.md

```diff
@@ -0,0 +1,162 @@
+---
+post_title: How to Update or Add a Registry Key Value with PowerShell
+username: tfl@psp.co.uk
+Categories: PowerShell
+tags: PowerShell, registry, provider
+Summary: How you can update or add registry keys or registry key value entries.
+---
+**Q:** I am having a problem trying to update the registry. I am using the New-ItemProperty cmdlet, but it fails if the registry key does not exist. I added the –Force parameter, but it still does not create the registry key. The error message says that it cannot find the path because it does not exist. Is there something I am not doing? I include my script so you can see what is going on. Help me, please?
```

Comment:
Please you line breaks at 100. Backtick cmdlet names and bold parameters (without the -).
```suggestion
**Q:** I am having a problem trying to update the registry. I am using the `New-ItemProperty`
cmdlet, but it fails if the registry key does not exist. I added the **Force** parameter, but it
still does not create the registry key. The error message says that it cannot find the path because
it does not exist. Is there something I am not doing? I include my script so you can see what is
going on. Help me, please?
```

Reply:
fixed in next commit!